src/terraform/install.sh

```diff
@@ -26,7 +26,7 @@ TERRAFORM_DOCS_SHA256="${TERRAFORM_DOCS_SHA256:-"automatic"}"
 
 TERRAFORM_GPG_KEY="72D7468F"
 TFLINT_GPG_KEY_URI="https://raw.githubusercontent.com/terraform-linters/tflint/master/8CE69160EB3F2FE9.key"
-GPG_KEY_SERVERS="keyserver hkp://keyserver.ubuntu.com:80
```

Comment:
Many folks have reported that the fetching GPG key from `hkp://keyserver.ubuntu.com:80` stuck for them and works with `hkp://keyserver.ubuntu.com`. 

https://github.com/devcontainers/features/issues/323